stmhal/boards/STM32F439/mpconfigboard.h

```diff
@@ -0,0 +1,87 @@
+#include STM32_HAL_H
```

Comment:
I have no idea what it's for, it must have come from the STM32F439-DISCO port that I used as a base to get started